### DISCUSSION TITLE
how to prevent [import 'virtual:uno.css'] syntax got unresolved problem?

### DISCUSSION THREAD

**Discussion by stalkercn** [unocss/unocss#2448] [Q&A]
I'm using eslint / eslint-plugin-import  for module import check, and
```typescript
import 'uno.css'
```
OR
```typescript
import 'virtual:uno.css'
```
both got eslint error:

> Unable to resolve path to module 'virtual:uno.css'. eslint ([import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md))

**Comment by rtpacks** [comment]
Hello, has anyone solved this problem?

**Reply by rtpacks** [reply]
I have temporarily resolved the problem.

```js
// .eslintrc.js
rules: { 
  'import/no-unresolved': [
    2,
    {
      ignore: ['^virtual:'],
    },
  ],
}
```

**Comment by cstaleror** [comment]
Use 
`// eslint-disable-next-line import/no-unresolved`
`import 'virtual:uno.css'`
